src/Nethermind/Nethermind.Consensus/Producers/TxPoolTxSource.cs

```diff
@@ -215,37 +217,86 @@ private static void ChooseBestBlobTransactions(
             in UInt256 baseFee,
             ArrayPoolList<Transaction> selectedBlobTxs)
         {
-            int size = leftoverCapacity + 1;
+            int maxSize = leftoverCapacity + 1;
             // The maximum total fee achievable with capacity
-            using ArrayPoolList<ulong> dpFeesPooled = new(capacity: size, count: size);
+            using ArrayPoolList<ulong> dpFeesPooled = new(capacity: maxSize, count: maxSize);
```

Comment:
what this does?

Reply:
Best discovered sum of priority fees at a given capacity, so far